src/Directory.Build.props

```diff
@@ -33,6 +33,9 @@
   <PropertyGroup Condition="$(MSBuildProjectName.Contains('.Test')) == true or $(MSBuildProjectName.Contains('.Sample')) == true">
     <IsPackable>false</IsPackable>
   </PropertyGroup>
+  <PropertyGroup Condition="$(MSBuildProjectName.Contains('.Test')) != true and $(MSBuildProjectName.Contains('.Sample')) != true">
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+  </PropertyGroup>
```

Comment:
Do we need this for every IDE build or does the `build.cmd pack` suffice?

Reply:
i think we should only do it with pack, i agree...
but i think the `.Where(x => x.ToString().Contains("NSwagStudio"))` is filtering out all projects which should be packed...

Reply:
what about adding a custom property to the projects which should be packed and then search them by this property?

Reply:
The problem was with loading projects, before getting the info it crashed trying to load wix reference. Generally IsPackable=false would be ideal, or vice versa not having anything packable and then allowing with property per project

Reply:
changed.